TESTS/psa/crypto_init/main.cpp

```diff
@@ -16,14 +16,15 @@
 * limitations under the License.
 */
 
-#if ((!defined(TARGET_PSA)) || (!defined(MBEDTLS_PSA_CRYPTO_C)))
-#error [NOT_SUPPORTED] Mbed Crypto is OFF - skipping.
+#if ((!defined(TARGET_PSA)) || (!defined(MBEDTLS_PSA_CRYPTO_C)) || (!defined(MBEDTLS_PSA_CRYPTO_SPM )))
+#error [NOT_SUPPORTED] Mbed SPM Crypto is OFF - skipping.
 #endif // TARGET_PSA
 
 #include "greentea-client/test_env.h"
 #include "unity/unity.h"
 #include "utest/utest.h"
 #include "crypto.h"
+#include "crypto_extra.h"
```

Comment:
crypto_extra should not need to be included explicitly by tests or user applications. It's already included by `crypto.h`.

Reply:
:+1: